spring-cloud-dataflow-tasklauncher/spring-cloud-dataflow-tasklauncher-function/pom.xml

```diff
@@ -17,6 +17,10 @@
 			<groupId>org.springframework.cloud</groupId>
 			<artifactId>spring-cloud-dataflow-rest-client</artifactId>
 		</dependency>
+		<dependency>
```

Comment:
Was it meant to add this if issue is just to update readme?

Reply:
Yes. That is needed to expose the client connection properties in metadata